tests/test_mappings/test_signals.py

```
@@ -84,3 +84,23 @@ def test_run_pre_mapping_settings_triggers(db, mocker, test_connection):
+
+
+@pytest.mark.django_db()
+def test_patch_integration_settings_on_card_mapping(test_connection, mocker):
+    """
+    Test patch_corporate_card_integration_settings is called when corporate card mapping is created
+    """
+    workspace_id = 3
+
+    mock_patch = mocker.patch('apps.mappings.signals.patch_corporate_card_integration_settings')
+    mapping = Mapping(source_type='CORPORATE_CARD', destination_type='CREDIT_CARD_ACCOUNT',
+                     source_id=7775, destination_id=123, workspace_id=workspace_id)
```

Flake8 (7.3.0)

[error] continuation line under-indented for visual indent

(E128)